src/components/EggheadPlayer/index.js

```diff
@@ -109,23 +103,78 @@ export default class EggheadPlayer extends Component {
   renderPlayer = (Player) => {
     const url = this.getUrl()
     const active = Player.canPlay(url)
-    const {...activeProps} = this.props
+    const {resource, ...activeProps} = this.props
     const props = active ? {...activeProps, ref: this.ref} : {}
     const {
-      onPlaybackRateChange,
       playbackRate,
-      onVolumeChange,
       volumeRate,
-    } = useEggheadPlayer()
```

Comment:
the `use` prefix is very specifically for **react hooks** and hooks can't be used in old school class components like this one!